static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -125,6 +102,16 @@ export const init = (): Promise<void> => {
             }
         );
 
+        onConsentNotification('advertisement', state => {
```

Comment:
I think it might have been nice to introduce this framework separately from wiring it into the Google tag configuration. If this bit causes problems we'll probably have to revert the whole thing rather than this specific integration point. It's good to have done this in development - doing/stubbing a few actual integrations is important for validating the approach. That doesn't mean they necessarily have to land together though.

These callbacks won't exist just yet so I guess it shouldn't be a big deal, but that's another reason why this probably isn't required at this stage.

Reply:
A good point, we should release the framework independently from the work required to wire it into these third party integrations. I'll talk with @ripecosta to see whether we can break this PR up.